packages/store/src/store.ts

```diff
@@ -43,7 +43,10 @@ export class Store {
       map(selectorFn),
       catchError(err => {
         // if error is TypeError we swallow it to prevent usual errors with property access
-        if (err instanceof TypeError) {
+        const suppressErrors: boolean | undefined = this._config.selectorOptions
```

Comment:
shouldn't be?

```TS
const { suppressErrors } = this._config.selectionOptions;
```

because the `NgxsConfig` tells that `suppressErrors: true // TODO: default is true in v3, will change in v4`, so it's `boolean` by default

Reply:
fixed